src/__tests__/load-modules.test.ts

```diff
@@ -15,12 +15,17 @@ describe('loadModules', function() {
     const container = createContainer()
 
     class SomeClass {}
+    class SomeNonDefaultClass {
+      static [RESOLVER] = {}
+    }
 
     const modules: any = {
       'nope.js': undefined,
       'standard.js': jest.fn(() => 42),
       'default.js': { default: jest.fn(() => 1337) },
-      'someClass.js': SomeClass
+      'someClass.js': SomeClass,
+      'someNonDefaultClass.js': { SomeNonDefaultClass },
```

Comment:
What should happen if there are multiple class exports in a file? Which one wins?

Reply:
Also, it might seem weird if you have a default export and also one marked with the symbol, which would win? Not saying anyone would have a reason to do this, but its an interesting question.

Reply:
In that case, the default one wins because it's the first one being checked. Again, we could check for all cases, and throw an error.

Reply:
It might seem like an odd limitation. But named exports always have a name, so perhaps we should use the name of the class/function instead of the file name for non-defaults, but you then risk registering way more than you bargained for, which was another reason I was against doing this.

Thoughts?

Reply:
Good point. In general, I'd say promoting accepted good practice should be the general guide here, e.g. one class/major function per file. In this case, limiting to one service per file.

Although, as I said above, I don't think it matters too much how much you allow as long as errors are thrown fast and hard. Determine the name like that:

- default export -> use file name
- named export having [RESOLVER] property
  - use name from inline resolver options
  - otherwise use constructor name
  - otherwise use file name

At every step, check if a service with that name already exists, and, if so, throw. As far as I can see, that should leave no room for unexpected overwriting.

Registering more than you bargained for should be hard if the [RESOLVER] property really is required for an export to be loaded, since it would have to be explicitly set on the export. Under the above logic, name collisions would not go unnoticed.

Reply:
> Then you have to extend the logic to check they all have an explicit name, which means more checks, etc.

Well, I think it would actually be simpler to just register every named export that defines the RESOLVER symbol, that way we don't have to throw an error if we just use the export name rather than the file name for those (file name would be for default export only)